action.yaml

```diff
@@ -10,16 +10,21 @@ inputs:
   results-dir:
     description: 'Save results to folder'
     required: false
-    default: 'qodana'
+    default: '/github/workspace/qodana'
   report-dir:
     description: 'Save html report to folder'
     required: false
-    default: 'report'
+    default: '/github/workspace/report'
+  cache-dir:
+    description: 'Save cache to folder'
+    required: false
+    default: '/github/home/qodana-cache'
 runs:
   using: 'docker'
-  image: 'docker://jetbrains/qodana:2020.3-eap'
+  image: 'docker://jetbrains/qodana:2021.1-eap'
```

Comment:
Maybe `:latest`?

Reply:
In this case if we release a new version which will change existing functionality, all user workflows will be affected